css/styles.css

```diff
@@ -444,7 +444,9 @@
   /* Contact form section */
   #foot {
     box-sizing: border-box;
-    background-image: url("../images/blue-background.png");
+    /* background-image: url("../images/blue-background.png"); */
```

Comment:
- [ ] [ OPTIONAL ] Kindly remove the commented-out code 👍🏽  because it makes your code prone to errors, hence harder to maintain 📁